### PR TITLE
Add oas3-api-snippet-enricher mention

### PR DESCRIPTION
**Pull request by andrzejwp**
Add oas3-api-snippet-enricher to list of documentation tools. The tool automatically includes code samples for all methods in the API.

### CODE REVIEW

**Comment by MikeRalphson** [comment]
Should support yaml to be OpenAPI compliant...

**Comment by andrzejwp** [comment]
YAML support added.

**Comment by MikeRalphson** [comment]
:+1:

File: _data/tools.yml [hunk 1]
```diff
@@ -149,6 +149,14 @@
   v2: true
   v3: true
 
+- name: oas3-api-snippet-enricher
+  category: documentation
+  language: JavaScript
+  github: https://github.com/cdwv/oas3-api-snippet-enricher/
+  link: https://github.com/cdwv/oas3-api-snippet-enricher/
+  description: Enrich your OpenAPI 3.0 JSON with code samples
```
Comment (philsturgeon) [inline _data/tools.yml]:
```suggestion
  description: Enrich your existing description documents with generated code samples
```

**Comment by andrzejwp** [comment]
Thanks Mike & Phil!

**Comment by andrzejwp** [comment]
Hi @philsturgeon @MikeRalphson - anything else to be done here?

**Pull request merged**